README.md

```diff
@@ -1 +1,128 @@
-# grid3_client_go
\ No newline at end of file
+# **grid3_go:**
+
+grid3_go is a go client created to interact with threefold grid. It should manage CRUD operations for deployments on the grid.
+
+- ## **Grid3_go flow:**
```

Comment:
Design docs should never be in the README.md, in the readme i expected introduction to the project, a simple example how to use the library and maybe the tests/coverage banners and maybe a section very down to show how to run the tests